src/posix/platform/radio.cpp

```diff
@@ -128,6 +130,16 @@ void platformRadioInit(otUrl *aRadioUrl)
 
     SuccessOrDie(sRadioSpinel.GetSpinelInterface().Init(radioUrl));
     sRadioSpinel.Init(resetRadio, restoreDataset);
+
+    if (urlFemLnaGain != nullptr)
+    {
+        long urlValue = strtol(urlFemLnaGain, nullptr, 0);
+
+        VerifyOrDie(INT8_MIN <= urlValue && urlValue <= INT8_MAX, OT_EXIT_INVALID_ARGUMENTS);
+        femLnaGain = static_cast<int8_t>(urlValue);
+    }
+
+    sRadioSpinel.SetFemLnaGain(femLnaGain);
```

Comment:
```suggestion
        sRadioSpinel.SetFemLnaGain(static_cast<int8_t>(urlValue));
    }
```

By default, LNA gain is set to 0. So I think there is no need to call this if it is not specified in url.

Reply:
Done.